app/content/profile/index.js

```diff
@@ -300,6 +305,55 @@ function Index() {
 
   return (
     <BottomSheetModalProvider>
+      <DialogComponent
+        title={"Edit Profile Pic"}
+        content={"Upload New Pic, or Remove Current Pic"}
```

Comment:
"Edit Profile Picture"
"Upload a new Profile Picture, or Remove the current Profile Picture"
Although using a dialog makes it look like an error lol, it would be better if it is a modal

Reply:
Done (I think), see updated screenshots.

Styling may need work.